### PR TITLE
Add set-up files

### PR DESCRIPTION
**Pull request by Leboroz**
# Add set-up files.
Added linters, readme, and a gitignore.

### CODE REVIEW

**Review by Leboroz — COMMENTED** [review]
Please make sure there is no linters error before the project is ready.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header id="header">
+      <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo">
```
Comment (Leboroz) [inline index.html:12]: I think you should avoid lines of code that are too long. It makes your code less clean.

**Review by Leboroz — COMMENTED** [review]
# Changes require ♻️ 

Please read my comments and make the required changes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+    </header>
```
Comment (Leboroz) [inline index.html:20]: You should leave a blank line between large blocks of code to make your code more readable.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    <div class="container">
```
Comment (Leboroz) [inline index.html:21]: I think you should add a blank to separate large blocks of code such as sections to make your code more readable.

File: index.html (hunk 1 above)
Comment (henatan99) [reply index.html:12]: - [ ] Well-written comment except it needs to be a little more polite. Like adding `I think` phrase in front of the sentence would make it better. :pray:

File: index.html (hunk 3 above)
Comment (henatan99) [reply index.html:21]: - [ ] Again, try to make it more polite and avoid a commanding tone. :pray: :+1:

**Comment by henatan99** [comment]
Hi @Leboroz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the your inline comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**